emailsig/templates/base.html

```diff
@@ -0,0 +1,14 @@
+<head>
+<!-- Latest compiled and minified CSS -->
+<link rel="stylesheet" href="https://maxcdn.bootstrapcdn.com/bootstrap/3.3.7/css/bootstrap.min.css" integrity="sha384-BVYiiSIFeK1dGmJRAkycuHAHRg32OmUcww7on3RYdg4Va+PmSTsz/K68vbdEjh4u" crossorigin="anonymous">
```

Comment:
As far as I can see you're pulling the whole Boostrap to use a single `.container` class. Highly inefficient

Reply:
@tjaniczek  also use the well  class :p

Reply:
A lot of email clients won't pull in any external files so I don't think we could do this anyway.

Reply:
@adamoliver its mainly used for the apps interface

Reply:
@adamoliver template works fine in gmail and mac client

Reply:
Unfortunately that only accounts for 34%.
https://emailclientmarketshare.com/